apps/src/music/player/interfaces/FunctionContext.ts

```diff
@@ -4,8 +4,8 @@
 export interface FunctionContext {
   /** Name of the function.  Possibly localized. */
   name: string;
-  /** ID of the block doing the function call. */
-  functionCallBlockId?: string;
+  /** Procedure ID of the function. */
+  procedureID?: string;
```

Comment:
super nit: `procedureId` for consistency?
Also worth mentioning that this is a blockly-specific ID?

Reply:
Will do in a follow-up!